test/integration/networks/networks_test.go

```diff
@@ -362,8 +338,7 @@ func TestNetworks(t *testing.T) {
 					}
 
 					for _, networkType := range []string{
-						"base",
-						"restricted",
+						"svpc",
```

Comment:
there is no need for a `for` that is goin to loop over a single value `svpc` the for need to be removed